test/support/test_index_helpers.rb

```diff
@@ -1,8 +1,9 @@
 class TestIndexHelpers
   AUXILIARY_INDEX_NAMES = ["page-traffic_test", "metasearch_test"].freeze
   INDEX_NAMES = %w(mainstream_test government_test).freeze
+  GOVUK_INDEX_NAME = "govuk_test".freeze
   DEFAULT_INDEX_NAME = INDEX_NAMES.first
-  ALL_TEST_INDEXES = (AUXILIARY_INDEX_NAMES + INDEX_NAMES).freeze
+  ALL_TEST_INDEXES = ([GOVUK_INDEX_NAME] + AUXILIARY_INDEX_NAMES + INDEX_NAMES).freeze
```

Comment:
This might have changed in the meantime, but adding a new index here used to be super expensive and slow down the tests a lot.